reprocess-mediator/package-metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "id": "reprocess-mediator",
+    "name": "Reprocess Mediator",
+    "description": "A mediator that allows for configuring and reprocess of transactions",
+    "type": "use-case",
+    "version": "1.4.2",
+    "dependencies": ["interoperability-layer-openhim"],
+    "environmentVariables": {
+      "TRUST_SELF_SIGNED": "true",
+      "OPENHIM_MEDIATOR_URL": "https://openhim-core:8080",
+      "OPENHIM_USERNAME": "root@openhim.org",
+      "OPENHIM_PASSWORD": "instant101",
+      "REGISTER_MEDIATOR": "true",
+      "REPROCESSOR_API_BASE_URL": "http://reprocess-mediator:3000",
+      "REPROCESS_CORE_VERSION": "latest",
+      "REPROCESS_UI_VERSION": "latest"
```

Comment:
we should pin these tags. Should be a specific version not latest

Reply:
@bradsawadye shouldn't we pin the default to latest? Since this is now configurable via environment variables. Refer to this for context https://github.com/jembi/platform/pull/297